pkg/systemd/overview-cards/cryptoPolicies.jsx

```diff
@@ -34,7 +34,13 @@ import "./cryptoPolicies.scss";
 
 const _ = cockpit.gettext;
 
-const displayProfileText = profile => profile === "FIPS" ? profile : profile.charAt(0) + profile.slice(1, profile.length).toLowerCase();
```

Comment:
So for the record, please drop this function completely and jsut show `profile` as-is. Thanks!

Reply:
So something like this `const displayProfileText = profile => profile;`

Reply:
No, just completely drop the function and all calls to it.

Reply:
`const displayedProfile = profile;` 
@martinpitt is this correct ?

Reply:
No -- please remove all instances of `displayProfileText`, and directly render `profile` in the component.

Reply:
For eg : instead of this` {displayProfileText(currentCryptoPolicy)}` something like `{currentCryptoPolicy}` and same in all cases where `displayProfileText` comes 
@martinpitt ?